java/src/hu/holyoil/controller/GameController.java

```diff
@@ -32,7 +33,14 @@
      */
     @Override
     public void Step() {
-        System.out.println("Stepping");
+        Logger.Log(this, "Steps" + (Main.isRandomEnabled ? "" : " - switched off!"));
+
+        if (Main.isRandomEnabled) {
```

Comment:
Randomness does not effect win and lose conditions.

Reply:
lol true